docs/creating_review_app.md

```diff
@@ -38,70 +17,65 @@ CircleCI will match the `review-app-` prefix and either:
 2. Update an existing review app using `update_review_app.sh`
 3. Once CI is complete follow [this step](https://github.com/artsy/force/blob/master/docs/creating_review_app.md#dns-setup) to setup DNS.
```

Comment:
Important to note here that DNS subdomain must exactly match review app name, otherwise authentication will not be allowed to complete: `awesome-feature.artsy.net` will authenticate, while any other subdomain will return client errors.

Reply:
Or likely makes more sense to sense to add to `Accessing the review app` step below.

Reply:
@eessex - There's a section further down specifically on DNS. I am adding this note to that section. What do you think?

Reply:
:+1: